cleo_sdk/CLEO.h

```diff
@@ -288,7 +288,7 @@ struct CRunningScript
 	bool IsExternal() const { return bIsExternal; }
 	bool IsMission() const { return bIsMission; }
 	bool IsCustom() const { return bIsCustom; } // is this CLEO Script?
-	const char* GetName() const { return Name; }
+	std::string GetName() const { auto str = std::string(Name, Name + 8); str.resize(strlen(str.c_str())); return str; } // make sure it is always null terminated
```

Comment:
can it break the 0AAA opcode?

Reply:
Seems it did in case of threads named with 8 characters.

Reply:
I remember @OrionSR brought this issue some time ago. Perhaps we should slow down and think more about desired behavior.

Maybe we should keep all the current logic and instead enforce the 8th byte of the name to be a null-terminator?

Reply:
I made GetName returning std::string. So even 8 characters will be packed into string and null terminated.
Opcode 03A3 (set script name) supports up to 8 characters and apparently not includes terminator in that case.